### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by FK78**
Resolves #4603, #4604, #4605, #4606, #4607, #4608, #4609, #4610, #4611, #4612, #4613, #4614, #4615, #4616, #4617, #4618, #4619, #4620, #4621, #4622, #4623, #4624

**Overall change:** _Merging dependabot PRs._

**Code changes:**

- _`package.json` and `package-lock.json` bumped._
- _Snapshots updated._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [] This PR requires manual testing

### CODE REVIEW

**Review by thekp — APPROVED** [review]
note this PR includes a breaking change for `psammead-rich-text-transforms`, should be tested if it's being used.

**Review by RayNjeri — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -61,34 +61,34 @@
...
     "@bbc/psammead-image-placeholder": "^1.2.21",
     "@bbc/psammead-inline-link": "^1.3.15",
     "@bbc/psammead-locales": "^3.0.1",
     "@bbc/psammead-media-indicator": "^2.6.11",
-    "@bbc/psammead-media-player": "^2.2.0",
+    "@bbc/psammead-media-player": "^2.2.1",
     "@bbc/psammead-navigation": "^3.0.5",
     "@bbc/psammead-paragraph": "^2.2.18",
-    "@bbc/psammead-rich-text-transforms": "^1.1.2",
+    "@bbc/psammead-rich-text-transforms": "^2.0.0",
```
Comment (chris-hinds) [inline package.json:78]: can you double check with the media pod that we are good to pull in this new major version please

**Review by chris-hinds — COMMENTED** [review]
Left one comment about a major version change.

Are the snapshot updates here expected?

File: package.json (hunk 1 above)
Comment (FK78) [reply package.json:78]: > Last update was breaking, but it's only being used on a page that isn't even live yet anyway so no real risk there.

**Comment by FK78** [comment]
> Left one comment about a major version change.
> 
> Are the snapshot updates here expected?

Snapshots are updating because of bumps, for example `@bbc/psammead-media-player`.

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**